Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -212,6 +214,22 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         }
     }
 
+    private func loadDefaultPlugins() {
+        if let disableDefaultPlugins = options?[kDisableDefaultPlugins] as? Bool, disableDefaultPlugins {
```

Comment:
If we're passing an Option called `kDisableDefaultPlugins` to Clappr I would assume that will disable every and each plugin.

Instead, this just doesn't load the `MediaControl` plugins. I think we can change this option name to represent what it really does.

Plus, the normal behavior in the architecture is to just override plugins that have the same name. This could be the solution... I'm not sure why we need to disable all plugins.

Reply:
Done.